sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -1185,7 +1166,7 @@ public BigtableWriteException(KV<ByteString, Iterable<Mutation>> record, Throwab
 
   static void validateTableExists(BigtableConfig config, PipelineOptions options) {
     if (config.getValidate()) {
-      String tableId = config.getTableId().get();
+      String tableId = checkNotNull(config.getTableId().get());
```

Comment:
.get() will throw an exception if its not accessible. You need to check that its accessible and skip the check if it isn't.

Reply:
Done.